app/views/about/index.html.erb

```diff
@@ -29,7 +32,8 @@
   <%= render partial: "data_assurance" %>
   <%= render partial: "staging_area" %>
   <%= render partial: "free_and_open_source" %>
-  
+  <% render partial: "trademark" %>
```

Comment:
can you find out why this won't be rendered? :)

Reply:
because you have a small typo here :)

it should be `<%= render` but you have `<% render` (missing the equals `=` sign)